example/styling/cielab.2.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Single layer | CARTO</title>
```

Comment:
wrong titles

Reply:
Yep, on all of them. However, for all the styling examples there is much copy-pasted boilerplate. We could use a JSON for all of these styling examples, right?